evaluator.go

```diff
@@ -98,3 +107,18 @@ func Evaluate(tape string, out io.Writer, opts ...EvaluatorOption) error {
 
 	return nil
 }
+
+func checkRequires(cmds []Command, out io.Writer, v VHS) {
+	allBinaries := make([]string, 0)
+	for _, cmd := range cmds {
+		if cmd.Type == REQUIRE {
+			fmt.Fprintln(out, cmd.Highlight(false))
+			allBinaries = append(allBinaries, cmd.Args)
+		}
+	}
+	requireCmd := Command{
+		Type: REQUIRE,
+		Args: strings.Join(allBinaries, ","),
+	}
+	requireCmd.Execute(&v)
```

Comment:
Instead of deceptive calling the Require command like this, can we implement the binary checking logic directly here? (as I commented earlier).

We won't need an `ExecRequire` function then.

Reply:
I've made it so that `ExecRequire` only looks at it's args and appends an error to the VHS struct.